modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -24,11 +24,23 @@ interface RenderedDialog<T extends Dialog.DialogData> {
   readonly instanceApi: Dialog.DialogInstanceApi<T>;
 }
 
+const getInlineDialogSizeClasses = (size: Dialog.DialogSize): string[] => {
+  switch (size) {
+    case 'large':
+      return [ 'tox-dialog-inline--width-lg' ];
+    case 'medium':
+      return [ 'tox-dialog--width-md' ];
+    default:
+      return [];
+  }
+};
```

Comment:
```suggestion
  switch (size) {
    case 'large':
      return Optional.some('tox-dialog-inline--width-lg');
    case 'medium':
      return Optional.some('tox-dialog--width-md');
    default:
      return Optional.none();
  }
};
```
Follow on from my previous comment.